### ISSUE TITLE
Config to place bin (app dir) at another place

### ISSUE THREAD

**Issue by kolobok55** [portapps/brave-portable#16]
I want to create several brave portable copies to have own configs, extension, etc in each. But I want to share app dir, cs it same for all and it can save quote of storage place.
Does any config to do it - run brave with app in another place?

**Comment by caspertone2003** [comment]
@kolobok55 
As I learned it, Crazy-max is providing a portable executable of brave-core, not providing customizations to brave-core (forks).
I am also interested is such approach as you point. It is in my todo list to check replacing files by hardlinks. See for example here
https://superuser.com/questions/909731/how-to-replace-all-duplicate-files-with-hard-links

I tried in the past with Chrome portable and it was a disaster. I cannot remember why, perhaps the wrong tool. But when I updated one copy in a folder, the other copies in other folders did not got updated at the same time, and hardlinks were not refreshed properly.
Let me know if it works for you

**Comment by kolobok55** [comment]
> @kolobok55
> As I learned it, Crazy-max is providing a portable executable of brave-core, not providing customizations to brave-core (forks).
> I am also interested is such approach as you point. It is in my todo list to check replacing files by hardlinks. See for example here
> https://superuser.com/questions/909731/how-to-replace-all-duplicate-files-with-hard-links
> 
> I tried in the past with Chrome portable and it was a disaster. I cannot remember why, perhaps the wrong tool. But when I updated one copy in a folder, the other copies in other folders did not got updated at the same time, and hardlinks were not refreshed properly.
> Let me know if it works for you

It so hard - tools for tools for tools ((
There is ini file in google chrome portable which have options

[GoogleChromePortable]
Google ChromeDirectory=**..\App\Chrome-bin**
; The main app directory
ProfileDirectory=profile
; The profile directory

But chrome doesn't store config, extension, from PC to PC...

**Comment by caspertone2003** [comment]
@crazy-max 

Wait a sec!
Perhaps what @kolobok55 is asking could be implemented via [appname]-portable.yml ?

**Comment by crazy-max** [comment]
@caspertone2003 @kolobok55 Yes indeed, I could implement this feature through the [config file](https://portapps.io/doc/configuration/). I keep you in touch.

**Comment by Metal0gic** [comment]
this app isn't functioning as a portable app without this IMO, I launch this app on different computers and I'm starting fresh each time. no syncing, no sessions no history, wallet funds don't transfer over, no plugins etc

**Comment by caspertone2003** [comment]
@Metal0gic 

I can confirm that 0.63.48-15 (x64) can be transfered from PC to PC (W10 to W10) smoothly.
Extensions are kept, as well as session cookies and tabs. I do not use syncing nor wallet, so I cannot comment.

USB tends to be very slow, I just copies the full directory in one PC to usb and from the usb to the HD of the other PC. Zero issues....

**Comment by Metal0gic** [comment]
I can confirm that does not work for me, on each computer I transferred it to it was starting fresh.  or using the settings of previously installed non portable versions of brave

**Comment by crazy-max** [comment]
@Metal0gic Please open a separate issue and follow this guide : https://portapps.io/doc/reporting-issue/
